src/components/settings-menu/SettingsSection.js

```diff
@@ -20,6 +20,7 @@ import {
   ListItemDivider,
 } from '../list';
 import { Emoji } from '../text';
+
```

Comment:
i dont like this but we can talk about it cc @osdnk

Reply:
For some reason without that space it wasn't finding the imports using alias but looks like now it's ok. 🤷‍♂️